### PR TITLE
Fixed transfer function according to new Lexis event

### PR DESCRIPTION
**Pull request by favreau**

### CODE REVIEW

File: plugins/extensions/plugins/ZeroEQPlugin.cpp [hunk 1]
```diff
@@ -176,23 +176,15 @@ void ZeroEQPlugin::_setupHTTPServer()
...
         std::bind( &ZeroEQPlugin::_spikesUpdated, this ));
     _remoteSpikes.registerSerializeCallback(
         std::bind( &ZeroEQPlugin::_requestSpikes, this ));
 
-    _httpServer->handle( _remoteColormap );
-    _remoteColormap.registerDeserializedCallback(
-        std::bind( &ZeroEQPlugin::_colormapUpdated, this ));
-    _remoteColormap.registerSerializeCallback(
-        std::bind( &ZeroEQPlugin::_requestColormap, this ));
+    _httpServer->handlePUT( _remoteMaterialLUT );
```
Comment (tribal-tec) [inline plugins/extensions/plugins/ZeroEQPlugin.cpp]: rename to _remoteColorMaps?

Comment (tribal-tec) [inline plugins/extensions/plugins/ZeroEQPlugin.cpp]: no support for GET?

File: plugins/extensions/plugins/ZeroEQPlugin.cpp [hunk 2]
```diff
@@ -455,54 +398,51 @@ bool ZeroEQPlugin::_requestLookupTable1D( )
...
-    _remoteColormap.setRange( { range[0], range[1] } );
-    return true;
+
+    auto& contributions = transferFunction.getContributions();
+    for( const auto& contribution: _remoteMaterialLUT.getContributionVector( ))
+        contributions.push_back( contribution );
+
+    Vector2f colormapRange = DEFAULT_TRANSFER_FUNCTION_RANGE;
+    const auto& range = _remoteMaterialLUT.getRange();
+    if( range[0] != range[1] )
```
Comment (tribal-tec) [inline plugins/extensions/plugins/ZeroEQPlugin.cpp]: why is this necessary?

File: plugins/extensions/plugins/ZeroEQPlugin.h [hunk 3]
```diff
@@ -137,15 +125,10 @@ class ZeroEQPlugin : public ExtensionPlugin
...
 
     /**
-     * @brief This method is called when the colomap is updated by a ZeroEQ event
-     */
-    void _colormapUpdated();
-
-    /**
-     * @brief This method is called when the colormap is requested by a ZeroEQ event
-     * @return True if the method was successful, false otherwise
+     * @brief This method is called when the material lookup table is updated by
```
Comment (tribal-tec) [inline plugins/extensions/plugins/ZeroEQPlugin.h]: colorMaps. No more material lookup table

File: plugins/extensions/plugins/ZeroEQPlugin.h [hunk 4]
```diff
@@ -137,15 +125,10 @@ class ZeroEQPlugin : public ExtensionPlugin
...
+     *        a ZeroEQ event
      */
-    bool _requestColormap();
+    void _materialLUTUpdated();
```
Comment (tribal-tec) [inline plugins/extensions/plugins/ZeroEQPlugin.h]: dito

File: plugins/extensions/plugins/ZeroEQPlugin.cpp (hunk 1 above)
Comment (favreau) [inline plugins/extensions/plugins/ZeroEQPlugin.cpp]: _remoteMaterialLUT is more consistent with other class member names I think. And no GET for now since the renderer is not the 'owner' of the colormap.

File: plugins/extensions/plugins/ZeroEQPlugin.cpp (hunk 2 above)
Comment (favreau) [inline plugins/extensions/plugins/ZeroEQPlugin.cpp]: It's not, it's a temporary hack. I removed it.

File: plugins/extensions/plugins/ZeroEQPlugin.cpp (hunk 1 above)
Comment (tribal-tec) [inline plugins/extensions/plugins/ZeroEQPlugin.cpp]: materialLUT was the old name. The consistency so far was to name the variables close to the class name.

Comment (favreau) [inline plugins/extensions/plugins/ZeroEQPlugin.cpp]: As discussed, kept only the PUT command, and removed the default values.
